### PR TITLE
[zh-cn]: create the translation of `declarativeNetRequest.updateSessionRules()` method

### PR DESCRIPTION
**Pull request by tianyeeT**
…onRules()` method

### CODE REVIEW

**Review by yin1999 — COMMENTED** [review]
同 https://github.com/mdn/translated-content/pull/21326#pullrequestreview-2122678459

File: files/zh-cn/mozilla/add-ons/webextensions/api/declarativenetrequest/updatesessionrules/index.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: declarativeNetRequest.updateSessionRules
+slug: Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/updateSessionRules
+l10n:
+  sourceCommit: 43e3ff826b7b755b05986c99ada75635c01c187c
+---
+
+{{AddonSidebar}}
+
+修改扩展的作用域规则集。首先移除 `options.removeRuleIds` 中列出的规则的 ID，然后添加 `options.addRules` 中列出的规则。请注意：
```
Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/declarativenetrequest/updatesessionrules/index.md:10]:
```suggestion
修改扩展的作用域的一组规则。首先移除 `options.removeRuleIds` 中列出的规则的 ID，然后添加 `options.addRules` 中列出的规则。请注意：
```

File: files/zh-cn/mozilla/add-ons/webextensions/api/declarativenetrequest/updatesessionrules/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+---
+title: declarativeNetRequest.updateSessionRules
+slug: Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/updateSessionRules
+l10n:
+  sourceCommit: 43e3ff826b7b755b05986c99ada75635c01c187c
+---
+
+{{AddonSidebar}}
+
+修改扩展的作用域的一组规则。首先移除 `options.removeRuleIds` 中列出的规则的 ID，然后添加 `options.addRules` 中列出的规则。请注意：
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/declarativenetrequest/updatesessionrules/index.md:10]: 这里是“scoped rule”，不是 rule for extension's scope

Comment (tianyeeT) [inline files/zh-cn/mozilla/add-ons/webextensions/api/declarativenetrequest/updatesessionrules/index.md:10]:
```suggestion
修改扩展的一组作用域规则。首先移除 `options.removeRuleIds` 中列出的规则的 ID，然后添加 `options.addRules` 中列出的规则。请注意：
```

**Review by yin1999 — APPROVED** [review]

**Pull request merged**